controllers/teams.js

```diff
@@ -1066,6 +1073,7 @@ router.get('/:teamId/members', async (req, res, next) => {
 				{
 					payload: {
 						userId: user.userId._id,
+						cantBeAdmin: userRolesContainsStudent,
```

Comment:
I think the reverse (`canBeAdmin: !userRolesContainsStudent`) would be more expressive here (and prevent double-negative in the future)